index.bs

```diff
@@ -737,6 +769,51 @@ authorizing an authenticator.
 </div>
 
 
+#### Construct the credential - `constructCredentialCallback` method #### {#constructCredentialCallback}
+
+This method's purpose is to map |credentialCreationData|'s [=items=] into a new {{PublicKeyCredential}} object associated
+with the caller's [=environment settings object/global object=], while running on the main [=event loop=]. See also
+[[!CREDENTIAL-MANAGEMENT-1]].
+
+This algorithm accepts one argument:
+
+    :   |global|
+    ::  Which must be its caller's [=current settings object=]'s [=environment settings object/global object=].
+
+When this method is invoked, the user agent MUST execute the following algorithm:
+
+1.  Let |attestationObject| be a new {{ArrayBuffer}}, created using |global|'s [=%ArrayBuffer%=], containing the
+    bytes of |credentialCreationData|.<code>[=attestationObjectResult=]</code>'s value.
```

Comment:
This `<code>` should be before `|credentialCreationData|`, right?

Reply:
yes, thx